src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.text.DecimalFormat;
+
+public abstract class Figure implements Quadrangular, Triangular {
+    protected static final String DECIMAL_FORMAT_PATTERN = "#.00";
```

Comment:
Please read the HW description. Figure has a state - all figures have color. 
So in Figure class you should only declare String color property